remove_en/remove.ftl

```diff
@@ -345,7 +345,6 @@ remove-modal-optout-text = If your request is blocked, or if for any reason you
 
 remove-about-faq-title = FAQ
 remove-about-faq-long-title = Frequently Asked Questions
-remove-faq-title = Data Removal Pilot FAQ
```

Comment:
I guess this is related to the comment above. Why did we remove this string and hardcode it in the template above?